src/main/java/task/TaskSchedulerRunner.java

```diff
@@ -6,12 +6,19 @@
 import task.core.TaskManager;
 
 /**
- * Created by ahmetkucuk on 3/10/18.
+ * Sets up a TaskManager and appropriate connections
```

Comment:
You could have more descriptive explanation here. For example:

1. Default DB connections are tried in every X secs until all connections succeed. Exception is thrown in every trial
2. TaskManager is started. In every 30 sec It creates new HEK puller task and Image puller task, creates JobRecord in Monitoring DB etc. 
3. Monitoring App is started, which creates an HTTP server to listen XXXX port. That app is providing list of all JobRecords created in DB.

You can be even more descriptive.